whatwareweb.md

```diff
@@ -49,15 +49,15 @@ I am going to build a simple stereo amplifier. This amplifier will be able to pl
 | DC jack |  https://www.amazon.com/dp/B07XSQP3CT/ |  $4.88  |
 | Input selector switch | https://www.amazon.com/dp/B07JM3M2B4/ |  $13.49  |
 | Speaker terminals | https://www.amazon.com/dp/B08YMZ88Z5/ |  $6.39  |
-| Amplifier board | https://www.amazon.com/dp/B07WC2TK58/ |  $7.09  |
+| Amplifier board | https://www.amazon.com/dp/B07WC2TK58/ |  $21.99  |
```

Comment:
hey, did you change your link? also, what's the reason for switching the amplifier board?

Reply:
Yes I did, the reason is that I ordered the original boards (it came in a pack of 2) and they were both broken. I have returned them and chosen a different one with hopefully better QC.

Reply:
ahhh ok cool!